DependencyInjection/Compiler/AddMapperInformationCompilerPass.php

```diff
@@ -64,5 +64,11 @@ public function process(ContainerBuilder $container)
                 $mapper->addMethodCall('addUnique', array($class, $field, $options));
             }
         }
+
+        foreach (DoctrineCollector::getInstance()->getOverrides() as $class => $overrides) {
```

Comment:
Dependency inversion principle. This should be injected. Not sure it is easy for a compiler pass though, so don't bother if you struggle